### PR TITLE
add patch for PyTorch 1.2.0 to use version of torchvision that is compatible with Python 1.2.0

### PR DESCRIPTION
**Pull request by branfosj**
Put in a fix for https://github.com/pytorch/pytorch/issues/26608

### CODE REVIEW

File: easybuild/easyconfigs/p/PyTorch/PyTorch-1.2.0_fix_torchvision_test.patch [hunk 1]
```diff
@@ -0,0 +1,36 @@
+pytorch #26608
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyTorch/PyTorch-1.2.0_fix_torchvision_test.patch]: @branfosj please include the full URL (and fix the patch checksum afterwards)?

Comment (branfosj) [reply easybuild/easyconfigs/p/PyTorch/PyTorch-1.2.0_fix_torchvision_test.patch]: done

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bber0501u03a.bb2.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4b144a40204a4555305fa4bbb61c7701 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
bear-pg0211u03a.bear.cluster - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a520d7752a85395f84d5398c5665d4b5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3105.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e7ac107678fb51d9bbf97780c83b03e7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 13 out of 13 (1 easyconfigs in this PR)
node2585.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2cc3f05e35e14d755c1ad3ed34a44f95 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/2ff69e81ba121083eae7948fc295a1ab for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2071.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/25490f92709546ff8ce515c08c01badd for a full test report.

**Comment by boegel** [comment]
Going in, thanks @branfosj!

**Comment by JackPerdue** [comment]
Thanks a lot to the two of you.  Very timely... I hit this bug this morning and followed the PyTorch bug report (after Googling the error) to here.  Well done!!!  Time for 4.0.1!!! :)

**Comment by boegel** [comment]
@JackPerdue `eb --from-pr 8986` is your friend...

**Pull request merged**